docs/contracts.rst

```diff
@@ -221,6 +221,71 @@ Most functions of `HashedMap` require not only a key, but also its index, ranked
         // this creates a deep copy of the map
         HashedMap<int, bool> mapCopy = new HashedMap(b);
 
+Library ``HashedSet``
+-----------------------
+
+The `HashedSet` library provides a set-like data structure.
+Unique values are hashed before being stored.
+Most functions of `HashedSet` require an index, ranked by the value's sha256 hash in ascending order.
+
+**Constructor**
+
+* ``HashedSet(bytes data)``
+  Create an instance of ``HashedSet`` with some initial data.
+
+    .. code-block:: solidity
+
+        HashedSet<bytes, int> set = new HashedSet<bytes, int>(b'');
```

Comment:
Only takes one type, not 2. Change everywhere.
`HashedSet<int> set = new HashedSet<int>(b'');`